### ISSUE TITLE
Where is source code socket

### ISSUE THREAD

**Issue by duythien** [Minds/minds#51]
Hello,
I have see your code have connect a socket such as http://dev.minds.io:8010, but I was not see where is your code

Could you adding a information for this

Thanks

**Comment by markharding** [comment]
We'll be release the socket repo soon, but it has some hard coded variables that need removing before we can make it public. Closing this as it's a duplicate of #21

**Comment by duythien** [comment]
Thanks
